examples/source-transformer-now/Cargo.toml

```diff
@@ -10,5 +10,5 @@ path = "src/main.rs"
 [dependencies]
 tonic = "0.9"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
-numaflow = { git = "https://github.com/numaproj/numaflow-rs.git", branch = "main" }
+numaflow = { path = "../../" }
```

Comment:
we need to do this for all examples

Reply:
We can copy the `Dockerfile` changes (rust compiler image, source for final image) and `.dockerignore` files from `map-cat` example to the `source-transformer-now` example also.